plugins/CoreHome/stylesheets/layout.less

```
@@ -552,7 +563,7 @@ nav {
 
   #root{
     #secondNavBar + .pageWrap {
-      margin-left: 0;
+      width: 100% !important;
```

```suggestion
      width: calc(~'100% - 18px') !important; // 100% - pageWrap margin
```

_(same problem as `max-width: 992px`)_